data/json/items/comestibles/mutagen.json

```diff
@@ -225,6 +225,24 @@
     "color": "light_gray",
     "use_action": { "type": "mutagen_iv", "mutation_category": "SPIDER" }
   },
+  {
+    "id": "iv_mutagen_snail",
+    "copy-from": "iv_mutagen_flavor",
+    "type": "COMESTIBLE",
+    "name": { "str": "snail serum" },
+    "description": "A super-concentrated mutagen with the consistency of snot.  You need a syringe to inject it… if you really want to?",
+    "color": "light_gray",
+    "use_action": { "type": "mutagen_iv", "mutation_category": "GASTROPOD" }
+  },
+  {
+    "id": "iv_mutagen_batrachian",
+    "copy-from": "iv_mutagen_flavor",
+    "type": "COMESTIBLE",
+    "name": { "str": "batrachian serum" },
+    "description": "A super-concentrated mutagen with that looks like pond water… are the shadows in it moving?  You need a syringe to inject it… if you really want to?",
```

Comment:
"A super-concentrated mutagen **with that looks**"
I think the "with" should be removed... Or maybe I'm just bad at English?

Reply:
Seconding, the with should be removed.